tests/subworkflows/create_test_yml.py

```diff
@@ -0,0 +1,95 @@
+import os
+from unittest import mock
+
```

Comment:
```suggestion
from pathlib import Path
```

Reply:
Got it, I'll fix that in the module tests as well then :+1: